duet/views/ad_cost_breakdowns.view.lkml

```diff
@@ -0,0 +1,124 @@
+include: "//looker-hub/revenue/views/ad_cost_breakdowns.view"
+
+view: +ad_cost_breakdowns {
+
+  dimension: campaign_name {
```

Comment:
Are these not present in the `looker-hub` view?

Reply:
I see them: https://github.com/mozilla/looker-hub/blob/main/revenue/views/ad_cost_breakdowns.view.lkml#L18

Reply:
They are there in the code, but when I went into looker itself I did not see them anywhere. When I put them in `spoke-default` they were then visible to me

Reply:
Do you mean the explore wasn't visible to you, or the fields in the explore weren't visible? If it's the former, you don't need these fields. If the latter, something is wrong with Looker

Reply:
Correct, I went to "Ad Cost Breakdowns" and the fields were not present at the time. This fixed it and I wanted to get something delivered, so I figured I'd return to it later.